packages/augur-ui/src/services/browser-mesh.ts

```diff
@@ -118,5 +120,6 @@ export async function createBrowserMesh(
   const mesh = new Mesh(meshConfig);
   mesh.onError(createBrowserMeshRestartFunction(meshConfig, web3Provider, zeroX, config));
   await mesh.startAsync();
+  zeroX.client.emit('ZeroX:State:Started');
```

Comment:
can you use a constant, so that there isn't a typo somewhere. 

`packages/augur-sdk/src/constants.ts`
SubscriptionEventName